app/views/bikes/_bike_org_table.html.haml

```diff
@@ -21,11 +21,11 @@
             - if bike.created_at.today?
               %span.hidden
                 = bike.created_at.strftime("%-m.%-d.%Y %-H.%M")
-              Today at #{bike.created_at.strftime("%-I %p")}
+              = t(".today_at", created_at: bike.created_at.strftime('%-I %p'))
```

Comment:
Anytime you see dates, please switch them to use the `convertTime` path.

Reply:
Just to clarify -- are you referring to a translation key or the css class? I'm seeing `time.format.convert_time` and `convertTime`, respectively

Reply:
If you put a `time.format.convert_time` inside of something with a class of `convertTime`, js (moment) converts the time to the user's time zone.

This is the best solution to time localization that I've seen or come up with.

Reply:
FYI: the localization js also converts time that is in timestamp format, which is useful because our API returns timestamps

Reply:
Cool. Only on occurrence in this batch, in app/views/bikes/pdf.html.haml

Reply:
Ooooh. well, in that case we can't use the `convertTime` path here, because we can't do js in the pdf.

... But we don't need to do this whole today time thing, since the pdf is only generated once. We can just use the `short_with_hour` format